document_page_project/models/__init__.py

```diff
@@ -1,2 +1,3 @@
-from . import project_project
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
Why you readd license in init files, I think is useless

Reply:
I saw a lot of people putting the license :S